### ISSUE TITLE
Strange Cursor Behavior in Entry Editor

### ISSUE THREAD

**Issue by gnorris-ethz** [JabRef/jabref#9050]
### JabRef version

5.7 (latest release)

### Operating system

macOS

### Details on version and operating system

JabRef 5.7--2022-08-05--73c111c (via homebrew), Mac OS X 12.5 x86_64, Java 18.0.1, JavaFX 18.0.1+2

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. Select an entry in the database and open in the entry editor by double-clicking
2. Try to delete a character using the delete key (the forward delete, e.g. fn + delete on a macbook keyboard)
3. Two characters will be deleted instead; sometimes the cursor also gets moved two positions to the left

**Comment by Siedlerchr** [comment]
Hi, I am using the latest development version on mac, but I am still on Big Sur with my MacBook and this works fine for me:
JabRef 5.8--2022-08-11--f06e564
Mac OS X 11.6.6 x86_64 
Java 18.0.1 
JavaFX 18.0.2+2

Do you autosave enabled?

**Comment by gnorris-ethz** [comment]
No, `localAutoSave` is `false`. As far as I recall, I did not change many of the preferences from their default values, although looking at the preferences table, I apparently have 96 preferences deviating from their default value.

**Comment by gnorris-ethz** [comment]
If I did the correct digging, then it seems that JabRef is just [using](https://github.com/JabRef/jabref/blob/bb011c9313367a28990ae213b3920fe6cd10d1dc/src/main/java/org/jabref/gui/fieldeditors/contextmenu/EditorContextAction.java) the [TextInputControl](https://openjfx.io/javadoc/18/javafx.controls/javafx/scene/control/TextInputControl.html) module from JavaFX to perform the text editing operations.

I found an [openjdk JavaFX bug](https://bugs.openjdk.org/browse/JDK-8248914) which is similar to the second part of my issue but does not mention the first part.

**Comment by Siedlerchr** [comment]
Thanks for digging into it. It might be a javafx thing related to the Mac version. If you are lucky it might be mixed by the next javafx release
